app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/Wishlist.php

```diff
@@ -227,6 +227,37 @@ protected function _setCollectionOrder($column)
         return $this;
     }
 
+    /**
+     * Retrieve current customer object
+     *
+     * @return Mage_Customer_Model_Customer
+     */
+    protected function _getCustomer()
+    {
+        return Mage::registry('current_customer');
+    }
```

Comment:
```suggestion
    protected function _getCustomer(): Mage_Customer_Model_Customer
    {
        return Mage::registry('current_customer');
    }
```

Reply:
This one is not possible as the method existed previously.